.github/workflows/update-changelog.yaml

```diff
@@ -0,0 +1,43 @@
+name: "Update Changelog"
+
+on:
+  workflow_call:
+  release:
+    types: [released]
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # Fetch entire history of repository to ensure relase date can be
+          # extracted from commit of the given tag.
```

Comment:
Are we sure this isn't overkill to do? Not sure if this eventually can lead to timeouts? Just checking, nothing I'm too concerned about.

Reply:
I personally never had an issue where checking out a bigger repo lead to timeouts. 

Instead of passing `0` to `fetch-depth` we could also pass a number of commits. Maybe we can give this a try with like 2 or 10 commits?
While testing the workflow I ran into issue due to the missing history. Maybe just fetching the last 2-10 commits fixes that problem.

Reply:
Sounds good.

Reply:
Nah, this doesn't seem to work: https://github.com/stefanzweifel/git-auto-commit-action-demo-app/runs/5266092592?check_suite_focus=true#step:4:37

Without the full history, the command will fail. :(

```shell
git log -1 --date=short --format=%ad 'v1.0.0'
# fatal: ambiguous argument 'v1.0.0': unknown revision or path not in the working tree.
```

My git knowledge is not that good enough to understand, how we could fetch the release date in another way. Might take this question up on Twitter.

Reply:
Let's give it a try with feting the entire history 👍